src/js/utils/date-utils.js

```diff
@@ -49,8 +49,10 @@
         var dateFormatEnum = Object.freeze({
             shortDate: 'll', // example: Jan 01, 2016
             longDate: 'LLLL', // example: Friday, January 01, 2016
-            time: 'LTS z', // example: Jan 01, 2016 15:30 UTC
-            defaultFormat: 'll HH[:]mm z' // example: Jan 01, 2016 15:30 UTC
+            time: 'LTS z', // example: 15:30 UTC
```

Comment:
Does `LTS` include seconds information as well? A quick glance at moment docs says it does, but it's not in the inline comment at all.